packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -10,38 +10,27 @@
         @click="$emit('restaurant-card-clicked')">
 
         <!-- background image -->
-        <div
+        <restaurant-image
+            v-if="!!imgUrl"
```

Comment:
```suggestion
            v-if="imgUrl"
```

It will give the same result but might be a bit cleaner.